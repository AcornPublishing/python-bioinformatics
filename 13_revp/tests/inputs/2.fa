>Rosalind_2002
GCGCTCCTGATTTAATACGACGAGACGACCAGCCCCAGCCGAGATTTGTGCTGATCCGGT
CAAGCAATCCCTTGAACACGTAGCGCGGACAGTGAGACCTAGGGCGCACGTGCAGGGTCG
ATACTCACTGAAAGGCGCAGACGGATAGCTGGGCTCCGTAATCGTGTACGCAGTCGCCGG
ATGGGCCTCACCAAGAATGACCTGTCACATCCTTCGCGAATGGCGCGGATTATCACTTAC
TACATGGTAAGACATGTTCCTAACCGGGGAGATGCGTGGCTACGCCTGAACTTTCGTCTC
TTTTGCGTGGTACCAGTGGAGGCGGTGATACCTGATATATGTTAGCGGTCCCAACGACTC
TGGGTTCGGACCTATGGCTGGCAAAATACATCAATTCTCTAGTTGGCGCCCGAGTAGGAT
ATATCCTTGCCGGTTATGAACACAGTTTCTACAAATTGGCTTACTGCGCTTTTGCGGCGT
GACGACTATGCCTTAGCATCGTATCGCTTCTGCTGACATGGTCGCGGTCTGTCGATATCA
TTAGGGAGATTATGCAATAACAACAAGTCGTACCGAGAGTTGCGTTATTACTCCAGGGTA
CGAGGCAGGTATCGCGTAAGATAAGCAGCCACACATACACCTTCCACACTAGAACGTCTC
CTTTCAGCGGAAATATGTCATCATCAGGGGTTAATGTTATGTGGGTCAACTCGATGGGCT
AACCCCGCTTTAAGATCCGCTATGGTCGCCTTAGCATCTGCTCAAGTTTCTGTGTGACAT
GGGCATTGGGGAAACCCGACATTGTGTAAACGAATGATCCGAGTGGGCGTGGATCATAAT
AT
