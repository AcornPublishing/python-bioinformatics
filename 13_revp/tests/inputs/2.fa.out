1 4
12 4
53 4
56 4
78 4
83 4
84 4
99 4
104 4
108 4
111 4
118 4
135 4
147 4
166 4
177 4
184 4
215 4
223 4
224 4
243 4
253 4
264 4
310 4
335 4
336 4
337 4
393 4
399 4
406 4
419 4
420 4
421 4
430 4
454 4
466 4
517 4
523 4
532 4
535 4
553 4
570 4
598 4
613 4
649 4
654 4
673 4
691 4
711 4
730 4
734 4
778 4
816 4
832 4
839 4
98 6
107 6
214 6
252 6
309 6
335 6
405 6
419 6
534 6
106 8
213 8
308 8
418 8
417 10
416 12
